docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,47 @@
+# Getting started
+
+In this tutorial, we'll walk you through the process of deploying the SMTP Integrator charm. We'll be deploying the charm on top of Kubernetes although a machine substrate could also be used.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed.
+
+## Deploy this charm
+
+The charm will the deployed in a new model named `smtp-tutorial`:
+
+```
+# Add the model
+juju add-model smtp-tutorial
+
+# Deploy the charm
+juju deploy smtp-integrator
+
+# Provide the mandatory configurations
+juju config smtp-integrator host=smtp.example.com
+
```

Comment:
Even if no charm supports the requirer side yet, we should show a dummy example of deploying another charm and relating to it

Reply:
And this example should highlight when you want to use `smtp` vs `smtp-legacy`

Reply:
The getting started tutorial doesn't include relating to other charms. We'd need an additional how-to for that to be consistent with other documentation. There aren't any charm currently supporting this relation as requirers, so it will have to wait